shared/attestationutil/attestation_utils.go

```diff
@@ -68,3 +74,103 @@ func AttestingIndices(bf bitfield.Bitfield, committee []uint64) []uint64 {
 	}
 	return indices
 }
+
+// VerifyIndexedAttestationSig this helper function performs the last part of the
+// spec indexed attestation validation starting at Verify aggregate signature
+// comment.
+//
+// Spec pseudocode definition:
```

Comment:
Can you update the `pseudocode definition`,  it doesn't reflect to the implementation below

Reply:
Updated it to latest spec @terencechain

Reply:
i mentioned in the comment above where the implementation starts